workers/generic-worker/artifacts_multiuser_test.go

```diff
@@ -54,6 +54,7 @@ func TestChainOfTrustUpload(t *testing.T) {
 	// Chain of trust is not allowed when running as current user
 	// since signing key cannot be secured
 	if config.RunTasksAsCurrentUser {
+		td.Scopes = append(td.Scopes, "generic-worker:run-tasks-as-current-user:"+td.ProvisionerID+"/"+td.WorkerType)
```

Comment:
I would prefer that we do this [here](https://github.com/taskcluster/taskcluster/blob/3249015448f795d30ebbc3c3304c3b6d86c39284/workers/generic-worker/helper_multiuser_test.go#L43) so that it does not need to be duplicated for every test.